bear-requirements.yaml

```diff
@@ -139,8 +139,8 @@ pip_requirements:
     version: ~=0.1.2
   eradicate:
     version: ~=0.1.6
-  giturlparse:
-    version: ~=0.9.1
+  git-url-parse:
```

Comment:
There was a problem with `giturlparse` ?

Should it be a separate issue?

Is it raised upstream?

Reply:
I am not sure if it is a problem with giturl parse, it works fine for bitbucket's git repositories
It's just that it breaks for bitbucket's hg urls for ssh style

(for git ssh, BitBucket uses : git@bitbucket.org:virresh/issueresolvinggit.git
 for hg ssh, BitBucket uses : ssh://hg@bitbucket.org/virresh/mercurialtest )

Reply:
Upstream issues & PRs were created.